lib/pages/RidePage.dart

```diff
@@ -225,16 +219,29 @@ class CancelRideButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: EdgeInsets.only(top: 22, bottom: 22),
-      child: RideAction(
-          text: 'Cancel Ride',
-          color: Colors.red,
-          icon: Icons.close,
-          action: () async {
-            Navigator.push(context,
-                MaterialPageRoute(builder: (context) => CancelRidePage(ride)));
-          }),
+    return Semantics(
```

Comment:
Right now it's just saying Button, do we want "Cancel Ride, Button"?

Reply:
That's strange, it reads "Cancel Ride, Button" with TalkBack. I'll move the button label out to the Semantics just to be safe

Reply:
It works now, thanks for changing